docs/kubectl.md

```diff
@@ -390,6 +390,7 @@ Additional help topics:
   kubectl run-container Run a particular image on the cluster.
   kubectl stop          Gracefully shutdown a resource
   kubectl expose        Take a replicated application and expose it as Kubernetes Service
+  kubectl label         Update the labels on a resource
```

Comment:
----- Original Message -----
> > @@ -390,6 +390,7 @@ Additional help topics:
> >    kubectl run-container Run a particular image on the cluster.
> >    kubectl stop          Gracefully shutdown a resource
> >    kubectl expose        Take a replicated application and expose it as
> >    Kubernetes Service
> > +  kubectl label         Update the labels on a resource
> 
> Changing labels is done by specifying the --overwrite flag
> 
> ```kubectl label pods foo baz=bar --overwrite ```
> 
> I hadn't thought about remove before, but now I'm thinking about:
> 
> ```kubectl label pods foo baz-```

I thought of this exact same thing for env vars, and a trailing minus seemed correct.